common-rest/src/test/java/org/uniprot/api/rest/output/header/HeaderFactoryTest.java

```diff
@@ -7,6 +7,7 @@
 import javax.servlet.http.HttpServletRequest;
 
 import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
```

Comment:
Good morning Talal,
I was looking these UnitTests and noticed that we are not covering the new context logic. I think we also need to mock "request.getRequestURI()", this way it will add the context prefix. Also frontend always wrap queries with brackets, for example: (gene:P53). for this example above the name would be uniprotkb__gene_P53__2023_05_18. How do you suggest we solve this problem? One option would be remove duplicated "_". This would cover other use cases as well. Let me know if you have any doubt.